changelog/2.24.0.md

```diff
@@ -0,0 +1,39 @@
+# v2.24.0-ck8s1
+
+Released 2024-02-08
+<!-- -->
+> [!IMPORTANT]
+> **Platform Administrator Notice(s)**
+> - The number of arguments required to run the `upgrade` command has changed, now you need to specify the specific cluster, `sc`, `wc`, or `both` to apply to both clusters as the command did previously. The `ck8s-kubespray` usage has been updated to clarify this.
+> - `ck8s-kubespray` commands no longer support using other prefixes besides `sc` or `wc`.
```

Comment:
What should users do if they do that though?
```suggestion
> - `ck8s-kubespray` commands no longer support using other prefixes besides `sc` or `wc`.
```

Reply:
Good question, I think that could've been clarified from the beginning (#330)

Maybe @anders-elastisys can answer this one

Reply:
I think they would have to rename their `{}-config/` folders to either `sc-config` or `wc-config` but I think it would be good to clarify that 😄

Reply:
@OlleLarsson PTAL, I added a comment about renaming the folders. For additional context, since [v2.20.0-ck8s5](https://github.com/elastisys/compliantkubernetes-kubespray/blob/v2.20.0-ck8s5/bin/upgrade.bash#L105) you could not run the `upgrade` command if you used different prefixes, the changes introduced here enforces this for all other `ck8s-kubespray` commands.